 <div style="display:inline-block; position:absolute; top:0; right:0; color: #230e3e; font:italic 1em serif;">$\LaTeX$<span style="display:inline-block; vertical-align:bottom">
    <a href="https://www.mathjax.org">
    <img style="height:1.5em" title="Powered by MathJax"
    src="https://www.mathjax.org/badge/mj_logo.png"
    border="0" alt="Powered by MathJax" >
</a></span></div>
<div style="display:inline-block; position:absolute; top:0; left:0; color: #230e3e; font:italic 1em serif;"><a href="https://www.julialang.org">
    <img style="height:2.25em; vertical-align:top" title="Powered by Julia"
    src="./figures/julia-logo-mask.png"
    border="0" alt="Powered by Julia" >
</a></div>
</a></div>$\require{cancel} \require{mhchem} \require{bbox} \require{color} \require{enclose} \require{AMSmath} \require{AMSsymbols}$

<header style="background-color:#e7e2ee; color:#461d7c; line-height:2em; padding-left:1em; padding-right:1em; bottom:0; float:bottom; text-align:right; float:right; font-family:serif; font-size:small;">Problem 1.023 ⋮ Moran, Shapiro, Boettner, Bailey, <i>Fundamentals of Engineering Thermodynamics</i>, 9th edition.</header>  

# Problem 1.023
A closed system consisting of 2 lb of a gas undergoes a process during which the relation between pressure and volume is $PV^n = \small \textsf{constant}$. The process begins with $P_1 = 25\, \small \textsf{lbf/in}^2$, $V_1 = 10\, \small \textsf{ft}^3$ and ends with $P_2 = 100\, \small \textsf{lbf/in}^2$ The value of n = 1.3. Determine the final volume, $V_2$, in $\small \textsf{ft}^3$, and determine the specific volume at states 1 and 2, in $\small \textsf{ft}^3/\textsf{lb}$.

<div class="btn-info">&nbsp;<div class="badge"></div><i class="fa fa-bomb"></i><strong> Sidebar: Thermo notes</strong> </div>
<div class="panel-info"><div class="panel-heading">
    

Processes where $\small PV^n = \textsf{constant}$ are called **polytropic processes**. They are very common; some examples are piston-cylinder devices like bicycle air pumps, engines, lungs, etc. You'll see them soon in Chapter 2.
</div></div>

In [1]:
# a little Julia housekeeping
using Unitful;       # use unitful.jl units package for Julia

In [2]:
# define these unit constants to make the units a bit more readable downstream
const ft = u"ft"
const psi = u"lbf/inch"
const lbm = u"lb";   # unit constants let us assign "lbm" to "lb" for less ambiguity  

## Given   

Closed system process: $PV^n = \small \textsf{constant}$   
1. $P_1 = 25$ psi; $V_1 = 10\, \small \textsf{ft}^3$  
1. $P_2 = 100$ psi
3. $m$ = 2 lb; $n$ = 1.3

In [3]:
# given
P₁ = 25psi
V₁ = 10ft^3

P₂ = 100psi

m = 2lbm
n = 1.3;

## Find  

 1. $V_2$ $\small \left[\textsf{ft}^3\right]$  
 2. $v_1 \small \left[ \textsf{ft}^3/\textsf{lb}\right]$  
 2. $v_2 \small \left[ \textsf{ft}^3/\textsf{lb}\right]$

## Assumptions  

There is no mention of gage pressure anywhere, so assume the pressures you are given are absolute pressures. 

<div class="panel-warning"><div class="panel-heading">$\small \textit{Always work using absolute pressures, and convert your output to gage pressure when necessary.}$</div></div>

## Get Organized

You must get organized before you start wildly writing equations and numbers. 
Generally, consider what you need to find, and mentally plot a path to get there from what you know.  

We are given a process in which  

$$PV^n = \small \textsf{constant},$$  

which translates to  

$$P_1{V_1}^n = P_2{V_2}^n.$$    

We need $V_2$. Conveniently, 

$$V_2 = V_1 \left(\frac{P_1}{P_2}\right)^{1/n}. \tag{1} \label{eq:V_2} \\[18pt]$$   

From here, we calculate $v_1$ and $v_2$ directly from the definition of specific volume  

$$v \equiv \frac{V}{m}. \tag{2} \label{eq:v}$$  


  From $\ref{eq:V_2}$




## Attack   

We're organized and calm and ready to find everything we're supposed to find.  

In [4]:
V₂ = V₁ * (P₁/P₂) ^ (1/n)    # from eq(1)
𝑣₁ = V₁ / m                  # from eq(2)
𝑣₂ = V₂ / m                  # from eq(2)
println("V₂ = ",V₂)
println("𝑣₁ = ",𝑣₁)
println("𝑣₂ = ",𝑣₂)

V₂ = 3.4425236279856737 ft^3
𝑣₁ = 5.0 ft^3 lb^-1
𝑣₂ = 1.7212618139928368 ft^3 lb^-1


$$ 
\bbox[#dff0d8, 0.5em, border:2.5pt solid #4cae4c]{
\color{#3c763d}
V_2 = 3.443\, \small \ce{ft}^3       \qquad 
v_1 = 5 \small  \ce{\frac{ft^3}{lb}}   \qquad
v_2 = 1.721 \small  \ce{\frac{ft^3}{lb}}
}
$$

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>